dev/visualize_transforms.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
```

Comment:
that seems super useful! why not exposing it in the doc + package?

Reply:
also, should it be part of another PR? (this one is already getting fairly large)

Reply:
Oh yes my bad: I started to develop #282 on the wrong branch. I will take care of this when merging.